crates/bevy_render/src/camera/projection.rs

```diff
@@ -161,17 +161,49 @@ pub enum ScalingMode {
     FixedHorizontal(f32),
 }
 
+/// Project a 3D space onto a 2D surface using parallel lines, i.e., objects have the same
+/// apparent size regardless of depth. The volume contained in the projection is called the view frustum.
+///
+/// If the window is smaller than the view frustum, some objects within the view frustum will not appear on the screen.
+///
+/// Note that the cross sectional area of the view frustum and the apparent size of objects are inversely proportional.
```

Comment:
I used "cross sectional area" here since just "size" or "area" is a bit ambiguous and might refer to volume/surface area.

Reply:
@xgbwei yup, that makes sense!